### ISSUE TITLE
Manage function (bottom right)

### ISSUE THREAD

**Issue by MikeBetts44** [aprildunnam/PowerApps#104]
Hi, even though the manage function is invisible to non admins, you can still tap on the blank space and the admin function appears. It needs to disable as well as go invisible to non admins?

**Comment by SaLe02** [comment]
Hi @MikeBetts44 , did you find a solution to disable the view completly for non admins?

**Comment by MikeBetts44** [comment]
No, I created a separate app with reduced pages permanent to manage for admin. Then removed that function. The app still has issues especially with language settings.
